app/assets/javascripts/discourse/models/post.js.es6

```diff
@@ -320,6 +320,16 @@ Post.reopenClass({
     });
   },
 
+  mergePosts(selectedPosts, selectedReplies) {
+    return Discourse.ajax("/posts/merge_posts", {
+      type: 'PUT',
+      data: {
+        post_ids: selectedPosts.map(function(p) { return p.get('id'); }),
+        reply_post_ids: selectedReplies.map(function(p) { return p.get('id'); })
+      }
+    });
```

Comment:
@tgxworld The client is updated. :smiley:  There's no error handling in the AJAX call since the other AJAX calls didn't have it either. Can you point me to an example of how it should be handled in Discourse?